lib/beaker_puppet_helpers/dsl.rb

```diff
@@ -224,5 +224,30 @@ def fact_on(host, name, opts = {})
     def fact(name, opts = {})
       fact_on(default, name, opts)
     end
+
+    # Show if bolt package is available
+    #
+    # @param [Beaker::Host] host
+    # @return Boolean True if package is available.
+    #
+    def bolt_supported(host = default)
```

Comment:
Should we indicate it's a boolean?
```suggestion
    def bolt_supported?(host = default)
```